src/System.Collections/tests/Performance/Perf.List.cs

```diff
@@ -0,0 +1,198 @@
+                    list.Contains(contained); list.Contains(contained); list.Contains(contained); list.Contains(contained);
+                    list.Contains(contained); list.Contains(contained); list.Contains(contained); list.Contains(contained);
+                }
+        }
+
+        [Benchmark]
+        public void ctor()
+        {
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                {
+                    new List<object>(); new List<object>(); new List<object>(); new List<object>(); new List<object>();
+                    new List<object>(); new List<object>(); new List<object>(); new List<object>(); new List<object>();
+                    new List<object>(); new List<object>(); new List<object>(); new List<object>(); new List<object>();
+                }
+        }
+
+        [Benchmark]
+        [MemberData("TestLists")]
+        public void ctor_IEnumerable(List<object> list)
+        {
+            var array = list.ToArray();
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    new List<object>(array);
+        }
+
+        [Benchmark]
+        [MemberData("TestLists")]
+        public void GetCount(List<object> list)
+        {
+            int temp;
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                {
+                    temp = list.Count; temp = list.Count; temp = list.Count; temp = list.Count; temp = list.Count;
+                    temp = list.Count; temp = list.Count; temp = list.Count; temp = list.Count; temp = list.Count;
+                    temp = list.Count; temp = list.Count; temp = list.Count; temp = list.Count; temp = list.Count;
+                }
+        }
+
+        [Benchmark]
+        [MemberData("TestLists")]
+        public void GetItem(List<object> list)
+        {
+            object temp;
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                {
+                    temp = list[50]; temp = list[50]; temp = list[50]; temp = list[50]; temp = list[50];
+                    temp = list[50]; temp = list[50]; temp = list[50]; temp = list[50]; temp = list[50];
+                    temp = list[50]; temp = list[50]; temp = list[50]; temp = list[50]; temp = list[50];
+                    temp = list[50]; temp = list[50]; temp = list[50]; temp = list[50]; temp = list[50];
+                }
+        }
+
+        [Benchmark]
+        [MemberData("TestLists")]
+        public void GetEnumerator(List<object> list)
+        {
+            IEnumerator temp;
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                {
+                    temp = list.GetEnumerator(); temp = list.GetEnumerator(); temp = list.GetEnumerator();
```

Comment:
Good catch; not intentional. Also, rather than have separate tests for GetEnumerator, GetCurrent, and MoveNext I think a single test with a foreach through the elements would be adequate.